tools/bcl-test-importer/BCLTestImporter/BCLTestProjectGenerator.cs

```diff
@@ -76,44 +76,43 @@ public enum WatchAppType {
+									"monotouch_Mono.CSharp_test.dll", 
+									"monotouch_System.IO.Compression.FileSystem_test.dll", 
+									"monotouch_System.IO.Compression_test.dll",
+									"monotouch_System.ServiceModel.Web_test.dll",
+									"monotouch_System.Json_test.dll",
+									"monotouch_System.ComponentModel.DataAnnotations_test.dll",
+									"monotouch_System.Xml.Linq_test.dll",
+									"monotouch_System.Xml_test.dll",
+									"monotouch_System.Transactions_test.dll",
+									"monotouch_System.Runtime.Serialization_test.dll",
+									"monotouch_System.Numerics_test.dll",
+									"monotouch_System.Core_test.dll",
+									"monotouch_Mono.Security_test.dll",}),
+			(name:"BCL tests group 2", assemblies: new[] { "monotouch_System.Data_test.dll", 
+									"monotouch_Mono.Data.Tds_test.dll",
+									"monotouch_System.Net.Http_test.dll",
+									"monotouch_System.ServiceModel_test.dll",
+									"monotouch_System.Web.Services_test.dll",
+									"monotouch_System_test.dll",
+									"monotouch_corlib_test.dll",}), 
 
 			// XUNIT TESTS 
 
-			(name:"SystemDataXunit", assemblies: new [] {"monotouch_System.Data_xunit-test.dll"}),
-			(name:"SystemJsonXunit", assemblies: new [] {"monotouch_System.Json_xunit-test.dll"}),
-			(name:"SystemNumericsXunit", assemblies: new [] {"monotouch_System.Numerics_xunit-test.dll"}),
-			(name:"SystemSecurityXunit", assemblies: new [] {"monotouch_System.Security_xunit-test.dll"}),
-			(name:"SystemLinqXunit", assemblies: new [] {"monotouch_System.Xml.Linq_xunit-test.dll"}),
-			(name:"SystemRuntimeCompilerServicesUnsafeXunit", assemblies: new [] {"monotouch_System.Runtime.CompilerServices.Unsafe_xunit-test.dll"}),
-			(name:"SystemComponentModelCompositionXunit", assemblies: new [] {"monotouch_System.ComponentModel.Composition_xunit-test.dll"}),
-			(name:"SystemCoreXunit", assemblies: new [] {"monotouch_System.Core_xunit-test.dll"}),
-			(name:"SystemRuntimeSerializationXunit", assemblies: new [] {"monotouch_System.Runtime.Serialization_xunit-test.dll"}),
-			(name:"SystemXmlXunit", assemblies: new [] {"monotouch_System.Xml_xunit-test.dll"}),
-			(name:"SystemXunit", assemblies: new [] {"monotouch_System_xunit-test.dll"}),
-			(name:"CorlibXunit", assemblies: new [] {"monotouch_corlib_xunit-test.dll"}),
-			(name:"MicrosoftCSharpXunit", assemblies: new [] {"monotouch_Microsoft.CSharp_xunit-test.dll"}),
+			(name:"BCL tests group 3", assemblies: new[] { "monotouch_System.Data_xunit-test.dll",
+									"monotouch_System.Json_xunit-test.dll",
+									"monotouch_System.Xml_xunit-test.dll",
+									"monotouch_System.ComponentModel.Composition_xunit-test.dll",
+									"monotouch_System.Xml.Linq_xunit-test.dll",
+									"monotouch_System.Security_xunit-test.dll",
+									"monotouch_System.Runtime.Serialization_xunit-test.dll",
+									"monotouch_System.Runtime.CompilerServices.Unsafe_xunit-test.dll", }),
+			(name:"BCL tests group 4", assemblies: new[] { "monotouch_System.Core_xunit-test.dll",
```

Comment:
A couple of really minor naming points:

* Maybe name the XUnit and Unit tests differently: `BCL tests group X (NUnit)` and `BCL tests group X (XUnit)`.
* I'd intuitively expect the most basic/important assemblies to use a lower number: that the mscorlib/System tests would be in group 1, and more peripheral assemblies in group >1.